dataflow/build.gradle

```diff
@@ -128,6 +128,7 @@ def testDataflowAnalysis(taskName, dirName, className, diff) {
 testDataflowAnalysis("busyExpressionTest", "busyexpr", "busyexpr.BusyExpression", true)
 testDataflowAnalysis("liveVariableTest", "live-variable", "livevar.LiveVariable", true)
 testDataflowAnalysis("issue3447Test", "issue3447", "livevar.LiveVariable", false)
+testDataflowAnalysis("reachingDefinitionsTest", "reachingdefinitions", "reachingdefinitions.ReachingDefinitions", true)
```

Comment:
Yes, I need to change it to the singular format. The reason why I used the plural format was I used the name from spa... But sure, I think it should be in sigular.

Reply:
Thanks! Yes, the reason why I chose the plural was because I used the section name in spa... now changed it in my local to keep consistency with others.